client/src/components/header/Header.jsx

```
@@ -1,7 +1,59 @@
+  const logoutHandler = async () => {
+    await authApi.logout();
+    dispatch(logout());
+  };
+
   return (
-    <div>Header</div>
-  )
+    <div className={styles.wrapper}>
+      <div className={styles.header}>
+        <div className={styles.flexBox}>
+          <Logo></Logo>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)